### DISCUSSION TITLE
ts-plugin errors

### DISCUSSION THREAD

**Discussion by virexcircle** [nuxt/nuxt#26041] [General]
I dont know what's going on, but the newly created nuxt app is throwing lots of errors. The app is working fine when you run it but im getting a lot of warnings. 
![image](https://github.com/nuxt/nuxt/assets/161726246/64558938-4b2b-4f77-a6f8-f8d9fa4814e5)

**Comment by AloisSeckar** [comment]
Today there was a breaking-change release of (former) Volar. Might be related?

https://github.com/vuejs/language-tools/blob/master/CHANGELOG.md

Extension was renamed to Vue - Official, current version is 2.0.2 and you shoud disable takeover mode and uninstall Typescript Vue Plugin
